From c7d2b841f46831b159ed715cc5f313129ba6fb10 Mon Sep 17 00:00:00 2001
From: Tobias Christiansen <tobyase@serenityos.org>
Date: Fri, 11 Mar 2022 19:01:35 +0100
Subject: [PATCH 3/4] Remove use of sbrk()

Again, for performance reporting the function `sbrk` is needed which we don't have. We just stub it out.
---
 src/sbrk.cc | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/sbrk.cc b/src/sbrk.cc
index 6e5292f..704d75c 100644
--- a/src/sbrk.cc
+++ b/src/sbrk.cc
@@ -34,9 +34,6 @@ extern uint64_t top_of_memory();
 uint64_t
 top_of_memory()
 {
-   if (sizeof(const void *) == 4)
-      return 0xFFFFFFFFULL & uint64_t(sbrk(0));
-   else
-      return uint64_t(sbrk(0));
+   return 0xFFFFFFFFULL;
 }
 
-- 
2.36.1

